docker-hub/cancel-downgrade.md

```diff
@@ -0,0 +1,41 @@
+---
+description: You can cancel or downgrade a private repository subscription through Docker Hub
+title: Cancel or downgrade your plan
+keywords: cancel, downgrade, private repo, private repository, Docker Hub, Hub
+---
+
+You can downgrade or cancel your private repository subscription plan through Docker Hub. Every user account is entitled to one free private repository (not applicable for organization accounts).
```

Comment:
I kinda feel that the statement in parenthesis might be better as a full sentence "This is not applicable for organization accounts" or as a note.

> **Note:** Organization accounts come with x accounts as a part of their plan.